assets/src/blocks/TakeActionBoxout/TakeActionBoxoutEditor.js

```diff
@@ -0,0 +1,292 @@
+  attributes,
+  isSelected,
+  setAttributes,
+}) => {
+  const {
+    take_action_page,
+    title: customTitle,
+    excerpt: customExcerpt,
+    link: customLink,
+    linkText: customLinkText,
+    newTab,
+    imageId: customImageId,
+    className,
+    stickyOnMobile,
+  } = attributes;
+
+  const {
+    loading,
+    actPageList,
+    title,
+    excerpt,
+    link,
+    linkText,
+    imageId,
+    imageUrl,
+    imageAlt,
+  } = useSelect(select => {
+    const args = {
+      per_page: -1,
+      sort_order: 'asc',
+      sort_column: 'post_title',
+      parent: window.p4_vars.take_action_page,
+      post_status: 'publish',
+    };
+
+    const actionsArgs = {
+      per_page: -1,
+      sort_order: 'asc',
+      sort_column: 'post_title',
+      post_status: 'publish',
+    };
+
+    // eslint-disable-next-line no-shadow
+    const actPageList = [].concat(
+      select('core').getEntityRecords('postType', 'page', args) || [],
+      select('core').getEntityRecords('postType', 'p4_action', actionsArgs) || []
+    ).sort((a, b) => {
+      if (a.title.raw === b.title.raw) {
+        return 0;
+      }
+      return a.title.raw > b.title.raw ? 1 : -1;
+    });
+    const actPage = actPageList.find(actPageFound => take_action_page === actPageFound.id);
+
+    // Because `useSelect` does an API call to fetch data, the actPageList will be empty the first time it's called.
+    // Or first few times.
+    if (take_action_page && !actPage) {
+      return {loading: true};
+    }
+    const actPageImageId = actPage?.featured_media;
+
+    const customImage = customImageId && select('core').getMedia(customImageId);
+    const customImageFromId = customImage?.source_url;
+
+    const title = !take_action_page ? customTitle : actPage.title.raw; // eslint-disable-line no-shadow
+    const excerpt = !take_action_page ? customExcerpt : actPage.excerpt.raw; // eslint-disable-line no-shadow
+    const link = !take_action_page ? customLink : actPage.link; // eslint-disable-line no-shadow
+
+    const linkText = !take_action_page ? customLinkText : actPage?.meta?.action_button_text || DEFAULT_BUTTON_TEXT; // eslint-disable-line no-shadow
+
+    const imageId = !take_action_page ? customImageId : actPageImageId; // eslint-disable-line no-shadow
+    const imageUrl = !take_action_page ? customImageFromId : select('core').getMedia(actPageImageId)?.source_url; // eslint-disable-line no-shadow
+    const imageAlt = !take_action_page ? customImage?.alt_text : ''; // eslint-disable-line no-shadow
+
+    return {
+      actPageList,
+      title,
+      excerpt,
+      link,
+      linkText,
+      imageId,
+      imageUrl,
+      imageAlt,
+    };
+  }, [take_action_page, customTitle, customExcerpt, customLink, customLinkText, customImageId]);
+
+  const takeActionPageSelected = take_action_page && parseInt(take_action_page) > 0;
+
+  if (loading || !actPageList.length) {
+    return __('Populating block\'s fields…', 'planet4-blocks-backend');
```

Comment:
Removed `// eslint-disable-next-line quotes,@wordpress/i18n-ellipsis` with
`Definition for rule '@wordpress/i18n-ellipsis' was not found`

Based on https://github.com/WordPress/gutenberg/blob/trunk/packages/eslint-plugin/docs/rules/i18n-ellipsis.md